integreat_cms/cms/templates/contacts/contact_list_row.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
+<tr class="border-t border-solid border-gray-200 hover:bg-gray-100">
+    <td class="pl-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.title }}
+        </a>
+    </td>
+    <td>
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.name }}
+        </a>
+    </td>
+    <td>
+        <a href="{% url 'edit_poi' region_slug=request.region.slug language_slug=request.region.default_language.slug poi_id=contact.poi.id %}"
+           class="text-blue-500 hover:underline">
+            {{ contact.poi }}
+        </a>
+    </td>
+    <td class="pr-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.email }}
+        </a>
+    </td>
+    <td class="pr-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.phone_number }}
+        </a>
+    </td>
+    <td class="pr-2">
+        <a href="{{ contact.website }}"
+           target="_blank"
+           rel="noopener noreferrer"
+           class="text-blue-500 hover:underline"
+           title="{% translate "To website" %}">
+            {{ contact.website }}
+        </a>
+    </td>
+    <td class="pl-2 pr-4 text-right min">
+        {% if perms.cms.change_contact %}
+            <a href="{% url 'copy_contact' contact_id=contact.id region_slug=request.region.slug %}"
+               title="{% translate "Copy contact" %}"
+               class="btn-icon">
+                <i icon-name="copy"></i>
+            </a>
+            <button title="{% translate "Archive contact" %}"
+                    class="confirmation-button btn-icon"
+                    data-confirmation-title="{{ archive_dialog_title }}"
+                    data-confirmation-subject="{{ contact.best_translation.title }}"
```

Comment:
```suggestion
                    data-confirmation-subject="{{ contact.name }}"
```
I've just noticed now 😅